Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG OPM_IMAGE=brew.registry.redhat.io/rh-osbs/openshift-ose-operator-registry-rhel9:v4.17
+FROM $OPM_IMAGE
+
+ARG COMMIT
+ARG INDEX_FILE=./auto-generated/released.yaml
+#This files will be copied twice but it is not possible to COPY if not empty
+COPY $INDEX_FILE /configs/netobserv-operator.yaml
```

Comment:
during the last release, there was an issue due to having our files directly in "configs", they should be in a subdirectory "netobserv-operator": see https://github.com/netobserv/network-observability-operator/pull/1201/files

Reply:
```suggestion
COPY $INDEX_FILE /configs/netobserv-operator/netobserv-operator.yaml
```